Client/Frontend/LoginManagement/Cells/LoginDetailCenteredTableViewCell.swift

```diff
@@ -0,0 +1,53 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Shared
+import Common
+
+class LoginDetailCenteredTableViewCell: ThemedTableViewCell, ReusableCell {
+    struct UX {
+        static let fontSize: CGFloat = 12
+        static let spacingTopBottom: CGFloat = 26
+        static let spacingLeadingTrailing: CGFloat = 16
+    }
+
+    lazy var centeredLabel: UILabel = .build { label in
```

Comment:
Ideally should be private so we don't access the UILabel directly, and we know what is getting modified on this cell with a clear interface. Can be done in another PR.